src/content/contributing/translation-program/faq/index.md

```diff
@@ -9,6 +9,18 @@ sidebar: true
 
 If you're new to the Translation Program and are hesitant to jump in, here are some FAQs that can help you begin. Use this guide to find answers to the most common queries.
 
+## Can I get compensated for translating ethereum.org? {#compensation}
```

Comment:
At second glance, I don't think this should be the top question.

Reply:
You're probably right. I added it here since it's by far the most common question we get asked. Feel free to move it wherever you think would be more appropriate.

Reply:
@minimalsm Where do you think we should place it? It wouldn't really make sense in the middle given the rest of the questions are all about the actual translation process, and the alternative is at the bottom or on another page. If we want to keep it on this page I think it's fine at the top for now, can always reevaluate and move it in the future.